scripts/buidl.sh

```diff
@@ -1,107 +1,110 @@
 #!/usr/bin/env bash
 # Script for building a host with injected json data
+# Dependencies: Nix package manager, nix-command 
```

Comment:
This comment should not exist -- it should be defined in the flake. It might be useful to use the environment variables as such: `NIX_EXPERIMENTAL_FEATURES="nix-command flakes"` among pinning the nix version by requiring Nix to be installed in buildInputs.

Reply:
Pinning the Nix version is a great idea. 

Setting the environment variable is essentially the same thing as adding it as a flag in the `nix_flags` array. 

https://github.com/ponkila/nixobolus/blob/d5c8710ce6ea6d01544414418c160521bbd5eebb/scripts/buidl.sh#L125-L132

Reply:
ok, this flag is a better idea